tests/tickets/17001/pasteobject.md

```diff
@@ -1,6 +1,6 @@
 @bender-tags: bug, 4.7.0, trac17001
 @bender-ui: collapsed
-@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, pastefromword, flash
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, pastefromword
```

Comment:
Actually, the entire test looks like we could remove it, but maybe @jacekbogdanski will have a different opinion.

Reply:
We still have a code which is checked by this manual test, see https://github.com/ckeditor/ckeditor4/commit/77664e89f
I think that we should rather change this test a bit instead of removing it:

1. Add `allowedContent: true` to the editor.
2. Add `sourcearea` and `htmlwriter` plugins.
3. Change test description to verify if there is `object` element in the source.

@KarolDawidziuk could you update the test?